IBX-10688: SiteAccess-aware Raptor connector configuration documented in Dev-doc (#3101)
ezsystems/developer-documentation: docs/cdp/raptor_integration/raptor_connector.md

---
description: Step-by-step activation procedure of setting up SiteAccess-aware Raptor connector.

> page_type: landing_page

Reviewer: ```suggestion
description: Step-by-step activation procedure of setting up SiteAccess-aware Raptor connector.
page_type: landing_page
month_change: true
```
Edit: I removed the LTS Update pill

---

> # SiteAccess-aware Raptor connector

Reviewer 2: ```suggestion
# Raptor connector
```
The fact that it's siteaccess-aware is not header-worthy, it's expected - and can be mentioned later.

The SiteAccess-aware [Raptor](https://www.raptorservices.com/) connector provides a seamless integration between [[= product_name =]] and Raptor Recommendation Engine.

Its primary goal is to enable editors and managers to deliver personalized experiences across digital channels, which helps to increase conversion rates, drive sales, and improve user engagement.

By combining content management capabilities with advanced recommendation features, the Connector allows teams to build and manage personalized experiences directly within a single platform.

This connector ensures a smooth and unified integration layer, enabling:

- event tracking through the tracking API
- personalized content delivery through the Recommendations API
- flexible, SiteAccess-aware configuration

This approach reduces integration complexity while providing a scalable foundation for personalization use cases across multiple sites and markets.

> Follow a step-by-step procedure that allows you to activate SiteAccess-aware Raptor connector.

Reviewer: And/Or there?

```suggestion
Follow a step-by-step procedure that allows you to activate SiteAccess-aware Raptor connector LTS Update.
```

> Follow a step-by-step procedure that allows you to activate SiteAccess-aware Raptor connector.

Reviewer 2: ```suggestion
Follow a step-by-step procedure that allows you to activate the Raptor connector.
```

> Activation includes configuration, adding tracking scripts and events, and using [Page Builder](page_builder_guide.md) blocks.

Reviewer 2: ```suggestion
To configure the Raptor integration. follow a step-by-step procedure that allows you to activate the Raptor connector.
Activation includes configuration, adding tracking scripts and events, and using [Page Builder](page_builder_guide.md) blocks.
```
This sentence feels out of place right now - and a link to the install/configuration doc should be added later.
